site/content/docs/user/quick-start.md

```diff
@@ -388,7 +388,7 @@ You can configure kind to use a proxy using one or more of the following [enviro
 * `NO_PROXY` or `no_proxy`
 
 **Note**: If you set a proxy it would be used for all the connection requests.
-It's important that you define what addresses doesn't need to be proxied with the NO_PROXY variable, typically you should avoid to proxy your docker network range `NO_PROXY=172.17.0.0/16`
+It's important that you define what addresses don't need to be proxied with the NO_PROXY variable, typically you should avoid to proxy your docker network range `NO_PROXY=172.17.0.0/16`
```

Comment:
aside: we may not need this cc @aojea

Reply:
indeed

https://github.com/kubernetes-sigs/kind/blob/1b4b217cfd74fc2181ce0d9888a80ed1a4cd0161/pkg/cluster/internal/providers/docker/provision.go#L288-L302